app/views/hostgroups/_form.html.erb

```diff
@@ -41,12 +41,17 @@
         <%= select_f f, :domain_id, accessible_domains, :id, :to_label, {:include_blank => blank_or_inherit_f(f, :domain)},
           {:help_inline => :indicator, :label => _("Domain"),
            :onchange => 'interface_domain_selected(this);', :'data-url' => method_path('domain_selected') } %>
-        <%= select_f f, :subnet_id, domain_subnets, :id, :to_label,
-                     { :include_blank => domain_subnets.any? ? blank_or_inherit_f(f, :subnet) : _("No subnets") },
-                     { :disabled => domain_subnets.empty?,
+        <%= select_f f, :subnet_id, domain_v4_subnets, :id, :to_label,
+                     { :include_blank => domain_v4_subnets.any? ? blank_or_inherit_f(f, :subnet) : _("No subnets") },
+                     { :disabled => domain_v4_subnets.empty?,
                        :data     => { :url     => SETTINGS[:unattended] ? freeip_subnets_path : '',
                                       :subnets => subnets_ipam_data.to_json },
-                       :label    => _("Subnet"),
+                       :label    => _("IPv4 Subnet"),
+                       :help_inline => :indicator } %>
+        <%= select_f f, :subnet6_id, domain_v6_subnets, :id, :to_label,
```

Comment:
I would suggest extracting this to a helper, then calling it for both methods. It's completely duplicate code and it can be easy to change something just on one side by mistake